### ISSUE TITLE
Can't find package in Atom

### ISSUE THREAD

**Issue by alexandernst** [leijou/php-server#4]
Searching for this package inside Atom returns no results.

**Comment by philippspinnler** [comment]
Same here +1

**Comment by cs278** [comment]
Probably because of this:

```
$ apm install php-server
Installing php-server to /home/chris/.atom/packages ✗
No available version compatible with the installed Atom version: 1.0.0
```

**Comment by leijou** [comment]
Should be fixed by PR #2
